### PR TITLE
Some fixes for the wrappers:

### PR DESCRIPTION
**Pull request by nbrinckm**
- saving of elements
- specification of outputs in the subclasses themselves
- some changes on the wps process class
- some addings on the mapping of the input parameters (complex outputs as inputs)

### CODE REVIEW

File: asyncwrapper/src/main/kotlin/org/n/riesgos/asyncwrapper/datamanagement/DatamanagementRepo.kt [hunk 1]
```diff
@@ -101,26 +102,29 @@ class DatamanagementRepo (
         var jobIdSetNotSetYet = true
         for (complexInputKey in complexInputs.keys) {
             val complexInput = complexInputs.get(complexInputKey) as ComplexInputConstraint
-            val searchForThisComplexInput = complexInputRepo.findByProcessWpsIdentifierInputWpsIdentifierLinkMimetypeXmlSchemaAndEncoding(
+            val searchForThisComplexInput = complexInputRepo.findByProcessWpsIdentifierJobStatusInputWpsIdentifierLinkMimetypeXmlSchemaAndEncoding(
```
Comment (MichaelLangbein) [inline asyncwrapper/src/main/kotlin/org/n/riesgos/asyncwrapper/datamanagement/DatamanagementRepo.kt:105]: Best. Method-name. Ever.

Comment (nbrinckm) [reply asyncwrapper/src/main/kotlin/org/n/riesgos/asyncwrapper/datamanagement/DatamanagementRepo.kt:109]: I know that it is quite verbose, but spring-data-jpa uses similar names (but those can create the method ifself just with a name like this). I wanted to be a little bit inline with the way spring-data-jpa or panache work.

**Comment by nbrinckm** [comment]
Which the changes that I have here it is possible to run deus succesfully by providing the constraints for shakyground, assetmaster & modelprop.

So which that merged, I would say we have a proof of concept done.

(Even that there are quite a lot of small things to improve further).

**Comment by nbrinckm** [comment]
I think with this here merged, we also can close https://github.com/riesgos/async/issues/24

**Pull request merged**